playwright/axe-test.ts

```diff
@@ -10,14 +10,11 @@ type AxeFixture = {
 // This new "test" can be used in multiple test files, and each of them will get
 // a consistently configured AxeBuilder instance.
 export const test = base.extend<AxeFixture>({
-  makeAxeBuilder: async (use) => {
+  makeAxeBuilder: async ({ page }, use) => {
```

Comment:
Are we clear to user typescript still? @rapidray12 @schmelz21 

Maybe fine since it's not the API just testing?

Reply:
The axe-core scripts are out of scope and at a minimum need to not run.  Unfortunately, there is no playwright python package for playwright accessibility tests. Node and Java will be our only options if we continue to incorporate these tests. 

@JCantu248 - I see the TODO's but what do we lose if we just remove this accessibility codebase for now?

Reply:
The Axe-Core stuff was a carry over when Playwright was run bare. I can remove the fixture here, but I'd also have to refactor all the test cases that utilize it.

Reply:
Ok. No need to refactor test then. I am good to move forward if all items are satisfied.